core/src/main/java/com/alibaba/nacos/core/cluster/lookup/AddressServerMemberLookup.java

```diff
@@ -43,30 +43,30 @@
  * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
  */
 public class AddressServerMemberLookup extends AbstractMemberLookup {
-    
+
```

Comment:
Do not change indent.
Please use nacos code style to format.
see src/style/codeStyle.md

Reply:
ok,copy that.